packages/shared/src/components/squads/utils.tsx

```diff
@@ -1,14 +1,8 @@
 import classed from '../../lib/classed';
 import { BasePageContainer, pageBorders } from '../utilities';
 
-export const SquadTitle = classed(
-  'h1',
-  'text-center typo-large-title font-bold',
-);
-export const SquadSubTitle = classed(
-  'p',
-  'text-center typo-title3 text-text-tertiary',
-);
+export const SquadTitle = classed('h1', 'typo-large-title font-bold');
+export const SquadSubTitle = classed('p', 'typo-body text-text-tertiary');
```

Comment:
Same comment here on Typography

Reply:
I think it looked a little verbose. Might be nice if we can come up with a function similar to this `classed` func but outputs a Typography.

Reply:
It would just be Typography?
I mean just omit this whole file and simply use `Typography` inside component?